src/program/open-spaces.njk

```diff
@@ -0,0 +1,173 @@
+                                        class="w-4 h-4 mr-2 mt-1">
+                                    <span><strong>For Organizers, By Organizers:</strong> A space for conference organizers, workshop leads, and community leads (like PyLadies, Django Girls, etc.) to share experiences and resources.</span>
+                                </p>
+                                <p class="flex items-start mb-3">
+                                    <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon"
+                                        class="w-4 h-4 mr-2 mt-1">
+                                    <span><strong>Preventing Burnout & Building Sustainable Habits:</strong> Open conversations about mental health, work-life balance, and community support.</span>
+                                </p>
+                                <p class="flex items-start mb-3">
+                                    <img src="{{ env.baseUrl }}img/assets/circle-purple.svg" alt="Bullet Icon"
+                                        class="w-4 h-4 mr-2 mt-1">
+                                    <span><strong>Recruitment & Career Development:</strong> Micro-workshops on hiring practices, resume reviews, or navigating tech careers.</span>
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="mb-12">
+            <div class="space-y-6 mb-8">
+                <div>
+                    <div
+                        class="black-han-sans-regular font-normal text-3xl md:text-5xl md:text-center text-[#000000] text-center lg:text-left">
+                        <p>Join the Community</p>
+                    </div>
+                    <div class="pt-4 text-md">
+                        <p class="mb-6 text-lg">Come with an idea, leave with new connections. Open Spaces are what you make them. Don't be shy to start the conversation!</p>
+                    </div>
+                </div>
+                <div class="mt-12">
+                    <div class="flex flex-col sm:flex-row gap-4 items-start">
+                        {{ button(text="Submit Your Idea",
+                        url="https://in.pycon.org/2025/open-spaces/",
+                        lime_bg=true, env =
+                        env) }}
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="grid md:grid-cols-2 gap-8 mb-24">
+            <div>
+                {% call flatCard(bg="purple") -%}
+                <div class="flex items-center mb-4">
+                    <img src="{{ env.baseUrl }}img/assets/circle-lime.svg" alt="Deadline Icon" class="w-6 h-6 mr-3">
+                    <h3 class="text-xl md:text-3xl">Deadline</h3>
+                </div>
+                <p class="text-2xl text-black">Submit your Open Space ideas by <strong>8th September,
```

Comment:
I am not sure what is the deadline in this, could you help me with that. The google doc does not have a deadline on till when can the ideas be submitted.